packages/x-components/src/x-modules/related-tags/components/related-tags.vue

```diff
@@ -16,9 +16,14 @@
         @slot Custom content that replaces the RelatedTag component.
         @binding {RelatedTag} relatedTag - Related tag data.
         @binding {boolean} highlightCurated - True if the curated RTs should be displayed.
+        @binding {string} itemClass - Class to add to the related tag.
        -->
-      <slot name="related-tag" v-bind="{ relatedTag, highlightCurated }">
-        <RelatedTag :highlightCurated="highlightCurated" :relatedTag="relatedTag">
+      <slot name="related-tag" v-bind="{ relatedTag, highlightCurated, itemClass }">
```

Comment:
Do we really need to bind `itemClass` here? 🤔 
I think that if the `related-tag` slot is used, there is no need to set the class using the prop.

Reply:
you are right @ajperezbau because you can "rewrite" the related-tag itself to assign the class